docs/virtual_machines/vms_and_vmis.md

```diff
@@ -0,0 +1,41 @@
+# Virtual Machine types
+There are two main virtual machine user facing API objects: a `VirtualMachine`,
+and a `VirtualMachineInstance`.
+
+## Virtual Machine Instance
+A
+[VirtualMachineInstance](https://kubevirt.io/user-guide/virtual_machines/virtual_machine_instances/)
+represents a single **running** virtual machine.
+
+It is encapsulated in a pod.
+
+When used without a controlling VMI object, it should be limited to running
+stateless workloads.
+
+Removing the VMI object represents stopping the Virtual Machine.
+
+## Virtual Machine
+A [VirtualMachine](http://kubevirt.io/api-reference/main/definitions.html#_v1_virtualmachine)
+in the other hand, represents a VM that is not running or in a stopped state.
+
+The `VirtualMachine` holds the template from which to create the VirtualMachineInstance.
```

Comment:
This seems like more introductory sentence than the one above.

Reply:
Good point. I guess I can also tie this sentence neatly to the one above. 

i.e. a VM is a template for running VMs (VMIs) which in reality represents a stopped VM. 

All "live" VM things are available on the corresponding VMI object.

Makes sense ? 
@VirrageS

Reply:
It makes sense. But I just wouldn't mention the "in reality represents a stopped vm". I feel like mentioning that VM is template for VMI which actually triggers creation of a Pod and in consequence a virtual machine is enough. 

I feel like there is no need to mention the state of the virtual machine. Unless just to point out that you can control the state of the VMI  (and effectively virtual machine) using VM spec.